src/App.js

```diff
@@ -1,26 +1,38 @@
 import React, { Component } from 'react';
-import './App.css';
+import {
+  BrowserRouter as Router,
+  Route,
+  NavLink,
+  Switch
+} from 'react-router-dom';
+
 import 'semantic-ui-css/semantic.min.css';
-import Home from './home/home'
-import { Menu } from 'semantic-ui-react'
+import { Menu } from 'semantic-ui-react';
 
-class App extends Component {
+import './App.css';
 
-  state = { activeItem: 'home' }
-  handleItemClick = (e, { name }) => this.setState({ activeItem: name })
+import Home from './home/home';
+import CitiesPage from './Cities/CitiesPage';
+import NotFound from './NotFound'
```

Comment:
I think consistent naming could help here. I like Home and NotFound so I suggest you rename CitiesPage to Cities. Furthermore, I think it's more common to have Capitalized names for compontents file too. So I suggest you you rename `home.js`.

One nice trick, suggestion. If you rename, for example, `./home/home.js` to `./Home/index.js` (without changing anything in the content), then you can do:

```javascript
import Home from './Home';
```